### PR TITLE
Print guide first version

### PR DESCRIPTION
**Pull request by arduibag**

### CODE REVIEW

File: content/guides/print-connection/introduction.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Follow the steps!
+
+1. [Who is your connector for?](step0-who-is-your-connector-for.html)  
+Understand your target user: Who will install it? Who will set it up? Who will use it? What are the needs of these stakeholders?
+2. [Who does what?](step1-who-does-what.html)  
+Understand how external stakeholders interact and synchronize their data with the PIM, and why.
+3. [Understand Akeneo PIM](step2-understand-akeneo-pim.html)  
+Understand Akeneo PIM, its features, its data structure in order to be able to manage these data in your print solution.
+4. [Reconcile PIM data with print capabilities](step3-reconcile-PIM-data-with-print-features.html)
```
Comment (NolwennP) [inline content/guides/print-connection/introduction.md:23]:
```suggestion
4. [Reconcile PIM data with print capabilities](step3-reconcile-PIM-data-with-print-features.html)  
```

File: content/guides/print-connection/step0-who-is-your-connector-for.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Let's already address 2 different needs:
+1. The need to create a complete print catalog from PIM information.
+2. The ability to create a PDF, a product brochure, a paper label or flyers on demand from one or more PIM products.
+
+
+### Julia's needs
+
+**For a print catalog:**
+
+Julia needs an easy way to export her PIM data into your print solution (Probably associated with a software such as **Adobe InDesign** or **Adobe Illustrator**).
```
Comment (NolwennP) [inline content/guides/print-connection/step0-who-is-your-connector-for.md:22]:
```suggestion
Julia needs an easy way to export her PIM data into your print solution (probably associated with software such as **Adobe InDesign** or **Adobe Illustrator**).
```

File: content/guides/print-connection/step0-who-is-your-connector-for.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+You will find below the main needs, associated to each user profile, in terms of process, documentation, actions related to your future print connector.
+
+Let's already address 2 different needs:
+1. The need to create a complete print catalog from PIM information.
+2. The ability to create a PDF, a product brochure, a paper label or flyers on demand from one or more PIM products.
+
+
+### Julia's needs
+
+**For a print catalog:**
```
Comment (NolwennP) [inline content/guides/print-connection/step0-who-is-your-connector-for.md:20]: Why is it in bold and on its own line?

File: content/guides/print-connection/step1-who-does-what.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+## Macro overview
+
+![Macro overview of the print connection](../../img/guides/print-connection-macro.svg)
+
+Akeneo PIM provides a single place to collect, manage, and enrich product information, create a product catalog, and distribute it to your print solution.
+
+In the context of a "PIM/print" architecture, the **PIM becomes the Reference of marketing product information**.
+
+
+## Akeneo PIM and DAM
```
Comment (NolwennP) [inline content/guides/print-connection/step1-who-does-what.md:12]: I don't see the link between this section and this step. Why do we need to talk about the PIM/DAM relation in the documentation for PIM/print connection?

File: content/guides/print-connection/step2-understand-akeneo-pim.md [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+Then, in order to understand our different editions and their capabilities, you can consult our [dedicated page](https://www.akeneo.com/compare-editions/) on our website.
+
+## Features and data model
+
+Play with our PIM!
+
+It is crucial to understand the PIM features and data model in order to create a great print connector.
+
+The best way to become an expert at Akeneo PIM is to play with it!  
+Here, you can access our [Akeneo PIM Community Edition demo platform](https://demo.akeneo.com) (login/password: admin / admin).
```
Comment (NolwennP) [inline content/guides/print-connection/step2-understand-akeneo-pim.md:22]: I think the demo is down.

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+* If you create a UI, add a “Test” button to test the connection (in order to check that everything is correctly configured on this side, and maybe to start retrieving some interesting information from Akeneo PIM)
+
+
+## Product management
+
+### Manage products
+
+![product mapping](../../img/guides/configuration-product.png)
+
+**What is it?**
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:33]:
```suggestion
**What is it?**

```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+
+* A UI or a “configuration file” to copy and paste Akeneo PIM API credentials (client id, secret, connection username and password).
+* If you create a UI, add a “Test” button to test the connection (in order to check that everything is correctly configured on this side, and maybe to start retrieving some interesting information from Akeneo PIM)
+
+
+## Product management
+
+### Manage products
+
+![product mapping](../../img/guides/configuration-product.png)
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:31]: I don't feel like this screenshot is helping.

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+
+::: warning
+The number of images that a catalog can contain can be quite large and we know that images dedicated to a print catalog can be very heavy (high definition).
+Make sure you have all the mechanisms in your connector to optimize the performance to manage image binaries.
+:::
+
+
+### Manage product associations
+
+**What is it?**
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:128]:
```suggestion
**What is it?**

```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+
+**What do you need to implement?**
+
+Well, here, it depends on your customers' needs.
+
+Also, don't forget our new feature that adds [quantities to PIM associations](https://help.akeneo.com/pim/serenity/updates/2020-07.html#new-association-type-with-quantities).
+
+### What about PIM product groups feature?
+
+**What is it?**
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:146]:
```suggestion
**What is it?**

```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+Groups are used to bring products together.
+In Akeneo PIM, you can group a selection of products to create a theme, for example a collection of products for  Christmas.
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:148]:
```suggestion
In Akeneo PIM, you can group a selection of products to create a theme, for example a collection of products for Christmas.
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+But most of the time, this feature is for internal purpose and product groups don’t have to be exported to your print solution.
+
+**What do you need to implement?**
```
Comment (NolwennP) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:151]:
```suggestion
**What do you need to implement?**

```

**Review by NolwennP — COMMENTED** [review]
3 images to remove because they don't seem to not be used anywhere:
- content/img/guides/erp-dam-pim-print.svg,
- content/img/guides/erp-pim-print.svg,
- content/img/guides/print-erp-connection-macro.svg.

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+Products are prepared in Akeneo PIM but not all of them are intended to be exported to your print platform.
+
+Because the PIM is multi-channels, multi-languages and offers the capability to model a catalog for different needs, you should allow your end-users to define which PIM data they want to import via an advanced filter system.
+
+Here are some examples of filters you should implement:
+
+* Channel: Only import product date related to the selected channel (most of the time a "print" channel).
+* Locale: Only import product data related to the selected locale.
+* Completeness: A PIM product can be exported if it is complete in terms of data contents, translation (localizable attribute) and information specific to the channels you are going to use (scopable attribute).
+* Enabled/Disabled status: Each PIM product has an enable/disable status. A PIM product can be exported if it has an enabled status only.
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:47]:
```suggestion
* Enabled/Disabled status: Each PIM product has an enable/disable status. You can set your connector to only export products that are `enabled`.
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+
+And sometimes, your customers may also need to import a PIM product into the print solution based on the following parameters:
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:49]:
```suggestion
And sometimes, your customers may also need to export a PIM product to the print solution based on the following parameters:
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 14]
```diff
@@ -0,0 +1,204 @@
...
+
+* Published product: some PIM customers can use this feature to work on different versions of the product. In the PIM, you can manage two different versions of a very same product: one “published” version that can be exported (using a published product export profile) and another version that is used to prepare the next product collection or season, for example. This is handled by the Akeneo PIM “Publication” feature.
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:51]:
```suggestion
* Published product:  In the PIM, you can choose to manage two different versions of the very same product: one “published” version that can be exported (using a published product export profile) and another version that is used to prepare the next product collection or launch, for example. This is handled by the Akeneo PIM “Publication” feature.
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 15]
```diff
@@ -0,0 +1,204 @@
...
+
+* The value of a specific attribute
+
+* A category tree or a subcategory tree…
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:55]:
```suggestion
* A category tree or a subcategory tree
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 16]
```diff
@@ -0,0 +1,204 @@
...
+...
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:56]:
```suggestion
* ...
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 17]
```diff
@@ -0,0 +1,204 @@
...
+
+**Conclusion:** each customer project has its own needs and it is important to implement enough flexibility in your connector settings to allow each customer to import only the product information he needs according to his criteria.
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:58]:
```suggestion
**Conclusion:** each customer project has its own requirements and it is important to offer enough flexibility in your connector settings to allow each customer to import only the product information they need according to their own criteria.
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 18]
```diff
@@ -0,0 +1,204 @@
...
+
+**What do you need to implement?**
+
+In the features of your connector configuration, you can add a “simple” and an “advanced” filter system to toggle from one to the other according to your customers' needs.
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:62]:
```suggestion
In the features of your connector configuration page, in order to be able to cover all of your customers' needs, you can choose to have one of the two filter systems. Either a “simple” one for straightforward use cases or an “advanced” one for the most specific requirements. 
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 19]
```diff
@@ -0,0 +1,204 @@
...
+
+### Product with variants
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:64]:
```suggestion
### Products with variants
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 20]
```diff
@@ -0,0 +1,204 @@
...
+
+**What is it?**
+
+Products with variants are products that have similarities, they are based on the same model but differ in some aspects from one another.
+
+Please consult [our documentation](https://help.akeneo.com/pim/v3/articles/what-about-products-variants.html), to learn everything about PIM product with variants:
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:70]:
```suggestion
Please consult [our documentation](https://help.akeneo.com/pim/v3/articles/what-about-products-variants.html), to learn everything about PIM products with variants:
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 21]
```diff
@@ -0,0 +1,204 @@
...
+
+::: info
+* PIM can manage up to 3 levels of enrichment (1 or 2 variant levels)
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:73]:
```suggestion
* Akeneo PIM can manage up to 3 levels of enrichment (1 or 2 variant levels)
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 22]
```diff
@@ -0,0 +1,204 @@
...
+  * 1 or 2 levels of “product model”
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:74]:
```suggestion
  * 1 or 2 levels of `product model`
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 23]
```diff
@@ -0,0 +1,204 @@
...
+  * 1 level of variant “product”
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:75]:
```suggestion
  * 1 level of variant `product`
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 24]
```diff
@@ -0,0 +1,204 @@
...
+* PIM variant axes can be:
+  * Simple select
+  * Reference entity single link (EE only)
+  * Measurement
+  * Boolean (Yes/No)
+:::
+
+**What do you need to implement?**
+
+Compared to your print solution and your end-user needs, you could offer a UI that allows your users to define how these variants will be displayed on the paper catalog page.
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:86]:
```suggestion
Compared to your print solution and your end-users' needs, you could offer a UI that allows your users to define how these variants will be displayed on the paper catalog page.
```

**Review by camillebourjade — COMMENTED** [review]
English Fix until 113

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 25]
```diff
@@ -0,0 +1,193 @@
...
+
+The main problem here is that Julia can use "Reference Entities" for a wide variety of needs.
+
+**What do you need to implement?**
+
+Well, two main use-cases here:
+
+1. **Manage Reference Entities as a specific print page for a set of products**
+
+As an entity reference can represent a product "universe" (ie: dedicated editorial content for a brand or a designer), you could enable your users:
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:184]:
```suggestion
As a reference entity can be used to add additional contextualized information around a given product (ie: dedicated editorial content for a brand or a designer), you could enable your users:
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 26]
```diff
@@ -0,0 +1,193 @@
...
+* to organize their products according to these reference entities
+* to create a specific header page to present this "universe" for a brand, a designer...
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:186]:
```suggestion
* to create a specific introduction page to present this "universe" for a brand, a designer, etc.
```

File: content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md [hunk 27]
```diff
@@ -0,0 +1,193 @@
...
+
+
+2. **Manage Reference Entities as rich information around a specific product attribute**
+
+Because a reference entity can be related to a specific attribute (ie: details of a color, a material...), you could enable your users:
+* to display this information at product level
```
Comment (camillebourjade) [inline content/guides/print-connection/step3-reconcile-PIM-data-with-print-features.md:192]:
```suggestion
* to display this information at the product level
```

File: content/guides/print-connection/step4-define-your-first-scope.md [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+The best way to interact with Julia's PIM is to develop a "[Connection-compliant](https://help.akeneo.com/pim/serenity/articles/what-is-a-connection.html)" connector (based on our API).
+
+:::tips
+Why?
+Please don't hesitate to read our ["4 reasons why you should use our API"](https://api.akeneo.com/documentation/why-the-api.html#4-reasons-why-you-should-use-our-api) page.
+:::
+
+## Follow the guide!
+
+As [mentioned before](step2-understand-akeneo-pim.html), there are [several editions](https://www.akeneo.com/compare-editions/) and several supported version of Akeneo PIM.
```
Comment (camillebourjade) [inline content/guides/print-connection/step4-define-your-first-scope.md:14]:
```suggestion
As [mentioned before](step2-understand-akeneo-pim.html), there are [several editions](https://www.akeneo.com/compare-editions/) and several supported versions of Akeneo PIM.
```

File: content/guides/print-connection/step4-define-your-first-scope.md [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+
+Of course, the choice of the PIM version/edition depends on your business but we strongly encourage you to develop a connector that is fully compatible with our latest **Serenity edition**.
```
Comment (camillebourjade) [inline content/guides/print-connection/step4-define-your-first-scope.md:16]:
```suggestion
Of course, the choice of the PIM version/edition depends on your business but we strongly encourage you to develop a connector that is fully compatible with our latest **SaaS edition**.
```

File: content/guides/print-connection/step4-define-your-first-scope.md [hunk 30]
```diff
@@ -0,0 +1,36 @@
...
+
+Well, now you can [start here](https://api.akeneo.com/getting-started/connect-the-pim-4x/welcome.html) to connect your print solution with our API! 🚀
+
+## Step by step
+
+**Our recommendation: always listen to your customer's needs first!**
+
+We are aware that it is difficult and expensive in terms of investment to achieve a 100% complete print connector that is flexible enough to adapt to any customer project.
+
+Furthermore, to build such a complex connector as a print connector in a single iteration would undoubtedly be too complex to specify.
```
Comment (camillebourjade) [inline content/guides/print-connection/step4-define-your-first-scope.md:30]:
```suggestion
Furthermore, to build such a complex connector as a print connector in a single iteration would undoubtedly be too complex to design from scratch.
```

File: content/guides/print-connection/step4-define-your-first-scope.md [hunk 31]
```diff
@@ -0,0 +1,36 @@
...
+
+**Our advice: adopt an Agile approach!**
+
+* Interview your future potential customers about their needs
+* Set priorities based on the most generic needs
```
Comment (camillebourjade) [inline content/guides/print-connection/step4-define-your-first-scope.md:35]:
```suggestion
* Set priorities based on the most common ones
```

File: tasks/build-doc.js [hunk 32]
```diff
@@ -432,6 +432,17 @@ gulp.task('build-guides', ['clean-dist','less'], function () {
               'step2-understand-akeneo-pim.md': 'Understand Akeneo PIM data',
               'step3-define-your-first-scope.md': 'How to build your connector'
             }
+        },
+        'print-connection': {
+            title: 'The complete guide to connect Akeneo PIM to your print solution',
```
Comment (camillebourjade) [inline tasks/build-doc.js:437]:
```suggestion
            title: 'A high-level guide to connecting Akeneo PIM to your print solution',
```

**Review by camillebourjade — COMMENTED** [review]
English fix

**Review by camillebourjade — APPROVED** [review]
English fix

**Pull request merged**